data/monitoring/prom-rule.yaml

```diff
@@ -19,3 +19,11 @@ spec:
           for: 5m
           labels:
             severity: warning
+        - alert: CnaoNotReady
+          annotations:
+            summary: One or more of CNAO deployed components is not ready.
+            runbook_url: http://kubevirt.io/monitoring/runbooks/CnaoNotReady
+          expr: sum(kubevirt_cnao_ready{namespace='{{ .Namespace }}'} or vector(0)) == 0
+          for: 5m
+          labels:
+            severity: warning
```

Comment:
runbook needed?

Reply:
Yes. Its needed.

Reply:
we have one https://github.com/kubevirt/monitoring/pull/19

Reply:
it will fail the expect otherwise..

Reply:
This answer belong to https://github.com/kubevirt/cluster-network-addons-operator/pull/944#discussion_r683205618
from what i understand.

In case its allowed that a metric name will be part of other metric name, then it should not Assert, and the code currently need to be updated to support this case.
In case its not allowed, then the code is fine, because it won't happen / find it out in case it happens, (depends on the other the metrics appear in the output)

Reply:
it could happen, so I changed it accordingly. thanks!